tfhe/c_api_tests/test_high_level_integers.c

```diff
@@ -64,6 +64,66 @@ int uint8_public_key(const ClientKey *client_key, const PublicKey *public_key) {
   return ok;
 }
 
+int uint8_safe_serialization(const ClientKey *client_key, const ServerKey* server_key) {
+  int ok;
+  CompactFheUint8List *lhs = NULL;
+  CompactFheUint8List *deserialized_lhs = NULL;
+  CompactFheUint8List *result = NULL;
+  Buffer value_buffer = {.pointer = NULL, .length = 0};
+  Buffer cks_buffer = {.pointer = NULL, .length = 0};
+  BufferView deser_view = {.pointer = NULL, .length = 0};
+  ClientKey *deserialized_client_key = NULL;
+
+  uint8_t lhs_clear = 123;
+
+  ok = client_key_serialize(client_key, &cks_buffer);
+  assert(ok == 0);
+
+  deser_view.pointer = cks_buffer.pointer;
+  deser_view.length = cks_buffer.length;
+  ok = client_key_deserialize(deser_view, &deserialized_client_key);
+  assert(ok == 0);
+
+  struct CompactPublicKey *public_key;
+
+  ok = compact_public_key_new(deserialized_client_key, &public_key);
+  assert(ok == 0);
+
+  ok = compact_fhe_uint8_list_try_encrypt_with_compact_public_key_u8(&lhs_clear, 1, public_key, &lhs);
+  assert(ok == 0);
+
+  ok = compact_fhe_uint8_list_safe_serialize(lhs, &value_buffer, 1 << 20);
+  assert(ok == 0);
+
+  deser_view.pointer = value_buffer.pointer;
+  deser_view.length = value_buffer.length;
+  ok = compact_fhe_uint8_list_safe_deserialize_conformant(deser_view, 1 << 20, server_key, &deserialized_lhs, 1);
```

Comment:
For clarity, I think these `1 << 20` should be put in a `const` like `const uint64_t max_serialization_size = UINT64_C(1) << UINT64_C(20);`